src/highlight.js

```diff
@@ -685,6 +685,14 @@ const HLJS = function(hljs) {
     return languages[name] || languages[aliases[name]];
   }
 
+  function registerAlias(name, names) {
+    if (typeof names === 'string') {
+      aliases[names] = name;
+    } else {
+      names.forEach(function(alias) { aliases[alias] = name; });
```

Comment:
```suggestion
  function registerAlias(alias, {languageName}) {
    let list = alias;
    if (typeof list === 'string') {
      list = [alias]
    }
    list.forEach(alias => aliases[alias] = languageName);
```

Reply:
Still don't love the naming, our global here is annoying when it competes for namespace with the arguments.

Reply:
Would you think it be better to pass a function to it too so that the API would have the same usage as `registerLanguage`, to prevent confusion?

~~~ .js
hljs.registerAlias(['jquery', 'jqueryui', function(hljs) {
    return {
        languageName: 'javascript'
    };
});
~~~

Reply:
No, that's because languages are executable code... no need for that complexity here.

Reply:
Just curious. What makes you decided to use `languageName` key and not `language`?

Reply:
They are two different concepts - and it causes all sorts of headaches elsewhere int he codebase when they are used interchangably.

- `language` <Language> the compiled language object returned by a grammar function
- `languageName` <string> "javascript", etc.

Reply:
Actually it's messier than that since "cpp" isn't a language name at all (the name is "C++"), it's an identifier... really just another alias!... but that's a problem for another day.

Reply:
Okay.